docs/proposals/2018-12-18 - Image Type Proposal.md

```diff
@@ -0,0 +1,100 @@
+# IMAGE TYPE PROPOSAL
+
+# Problem
+
+Anglerfish users want to categorize images by type, so that they can be queried in search, and have
+different fields be visible/required based on this type.
+
+# Proposal
+
+Add an `image_type` trait, represented as a machine-readable enumeration of the types: `["photograph", "graphic", "illustration", "thumbnail"]`
+
+## Photograph
+
+### Definition
+
+A capture of real life.
+
+### Rights and Usage
+
+A large proportion of photographs are not owned by the customer. If owned by the customer, indefinitely. If not, third-party is usually not generous because context in which image is shown is critical as well as proper attribution to the image's owner and distributor. However photos can also come from stock image providers who permit the image to be used in any context. These images are often one-time use only, but customers can also have contracts with news organizations to use the image indefinitely.
+
+## Graphic
+
+### Definition
+
+A work created to explain information.
+
+### Rights and Usage
+
+Depends on how large customer's graphics staff is. If large, then most infographics are owned by the customer. If owned by the customer, indefinitely. If not, third-party is usually not generous.
+
+## Illustration
+
+### Definition
+
+A creative work made for expression.
+
+### Rights and Usage
+
+Most infographics are owned by the customer. If owned by the customer, indefinitely. If not, third-party is usually not generous.
+
+## Thumbnail
+
+### Definition
+
+The promotional image for a video that appears before video is played to draw audience to consume the video.
+
+### Rights and Usage
+
+Depends on how large customer's video staff is. If large, then most infographics are owned by the customer. If owned by the customer, indefinitely. If not, third-party is usually not generous.
+
+
+# Trait in ANS Schema
+
+```
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "https://raw.githubusercontent.com/washingtonpost/ans-schema/master/src/main/resources/schema/ans/0.8.1/traits/trait_image_type.json",
+  "title": "Image Type",
+  "description": "A more specific category for an image.",
+  "type": "string",
+  "category": {
+    "type": "string",
+    "enum": [ "photograph", "graphic", "illustration", "thumbnail" ],
```

Comment:
If an image doesn't fit a category – though I'm not sure what that would be – will that element field not include this trait?

Reply:
Yup, the field will just not exist. For search purposes, they'll be considered Photographs though. That way all old content will still appear in searches by image_type, when type is Photograph.